examples/models/models_billboard.c

```diff
@@ -59,8 +80,15 @@ int main(void)
 
                 DrawGrid(10, 1.0f);        // Draw a grid
 
-                DrawBillboard(camera, bill, billPosition, 2.0f, WHITE);
-
+                // Draw order matters!
+                if (distanceStatic > distanceRotating) {
```

Comment:
Please, could you review [coding conventions](https://github.com/raysan5/raylib/blob/master/CONVENTIONS.md), raylib uses aligned brackets.

Reply:
Sorry, I missed this one. It should follow the coding conventions now.

Reply:
Excuse my insistence but same issu with the else, it should be
```c
if ()
{

}
else
{

}
```

Reply:
No, you're right. I don't usually write in this style and can't spot all of them. I should set up a linter. Sorry for wasting your time.

Reply:
No worries! no time wasted! 👍🙂